CONTRIBUTING.md

```diff
@@ -73,4 +73,61 @@ guidelines we follow:
   contribution. Finally, no branch merge commits should get into the
   *master* history.
 
+When developing Fencer, it is important to keep in mind that linting
```

Comment:
This should be a separate subsection to make it easier to link to it (Github allows linking to any header in a Markdown file).

Reply:
Ok, I'm putting it as a subsection of the Style Guide section.